server/src/main/java/org/apache/druid/metadata/SQLMetadataStorageActionHandler.java

```diff
@@ -357,7 +357,7 @@ private String getWhereClauseForActiveStatusesQuery(String dataSource)
         task = objectMapper.readValue(resultSet.getBytes("payload"), entryType);
       }
       catch (IOException e) {
-        log.error(e, "Encountered exception while deserializing task payload, setting task to null");
+        log.warn("Encountered exception[%s] while deserializing task payload, setting payload to null", e.getMessage());
```

Comment:
I think it should be "setting task to null" as in the next line we have `task = null;`

Reply:
Hmm, I'm not sure what the best term is here. If `task` is set to null, all fields of a complete task entry in the overlord UI would be filled with nulls. People might look at the logs why they're nulls and will see this log. But then, `Setting task to null` is not very intuitive because they will be probably not familiar with internal code of Druid. I used `payload` because it's the column name of metadata store table. Would it be more clear if it says `Encountered exception[%s] while deserializing task payload from the metadata store, setting payload to null`?

Reply:
Oh I didn't read the code before this log, yeah payload is correct, i got confused by the variable "task".